config/rubocop_base.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  NewCops: enable
```

Comment:
this is the thing that I have a conceptual issue with. this way, if you update rubocop in the project it automatically enables new cops, meaning you will have different linting results between projects depending on their rubocop version. This should be a whitelist of things instead.

Reply:
Agreed. Changed.